src/scripts/components/leaderboard/leaderboard-interface.js

```diff
@@ -0,0 +1,56 @@
+import leaderboardActions from './leaderboard-actions.js';
+
+class LeaderboardInterface {
+  form = document.querySelector('form');
+
+  refresh = document.querySelector('.refresh-btn');
```

Comment:
**Kindly make a few changes:**

- Implement the "Refresh" button (receiving data from the API and parsing the JSON). 
(Your refresh button isn't working).

- Fix the error issue that appears in the chrome console:
``
Uncaught (in promise) TypeError: Cannot read property 'inline' of undefined
``

Reply:
hello @carlos-ssh! the refresh button is working, but there is nothing to refresh when you submit data. (the scores already are on the leaderboard). But how is it supposed to work? are we supposed to keep one static id for our game? or are we supposed to create a new game every time we refresh the page?

Reply:
Was not able to reproduce the error, I tried in all browsers.